dockers/ubuntu-cuda/Dockerfile

```
@@ -160,7 +160,7 @@ RUN \
         # installing pytorch from wheels \
         CUDA_VERSION_MM=${CUDA_VERSION%.*} && \
         TORCH_VERSION_MM=${TORCH_VERSION%.*} && \
-        pip install -U "nvfuser-cu${CUDA_VERSION_MM/./}-torch${TORCH_VERSION_MM/./}" ; \
+        pip install -U --extra-index-url https://pypi.nvidia.com --pre "nvfuser-cu${CUDA_VERSION_MM/./}-torch${TORCH_VERSION_MM/./}" ; \
```

```suggestion
        pip install -U --extra-index-url https://pypi.nvidia.com "nvfuser-cu${CUDA_VERSION_MM/./}-torch${TORCH_VERSION_MM/./}" ; \
```
Is the Pre needed? Does this mean you are using unstable versions?

---

nvfuser only has pre-releases.